### PR TITLE
[stable/wordpress] Add custom labels on wordpress ingress

### PR DESCRIPTION
**Pull request by wjeanneau**
Implement custom labels on wordpress ingress, following the per host implementation.

### CODE REVIEW

File: stable/wordpress/README.md [hunk 1]
```diff
@@ -86,6 +86,7 @@ The following table lists the configurable parameters of the WordPress chart and
 | `ingress.hosts[0].tls`               | Utilize TLS backend in ingress             | `false`                                                    |
 | `ingress.hosts[0].tlsSecret`         | TLS Secret (certificates)                  | `wordpress.local-tls-secret`                               |
 | `ingress.hosts[0].annotations`       | Annotations for this host's ingress record | `[]`                                                       |
+| `ingress.hosts[0].labels`            | Labels for this host's ingress records     | `{}`                                                       |
```
Comment (juan131) [inline stable/wordpress/README.md]: I'd state very clear that they are "extra labels". You're not modifying the default ones:
```
     app:
     chart:
     release
     heritage:
```

File: stable/wordpress/templates/ingress.yaml [hunk 2]
```diff
@@ -9,6 +9,9 @@ metadata:
     chart: "{{ $.Chart.Name }}-{{ $.Chart.Version }}"
     release: "{{ $.Release.Name }}"
     heritage: "{{ $.Release.Service }}"
+    {{- if .labels }}
```
Comment (juan131) [inline stable/wordpress/templates/ingress.yaml]: It should be: 

```
    {{- if .Values.ingress.labels }}
+{{ toYaml Values.ingress.labels | indent 4 }}
    {{- end }}
```

**Review by juan131 — CHANGES_REQUESTED** [review]

File: stable/wordpress/templates/ingress.yaml (hunk 2 above)
Comment (wjeanneau) [reply stable/wordpress/templates/ingress.yaml]: Yes this is more a standard solution, but in this case we have a loop (**{{- range .Values.ingress.hosts }}**) to create a dedicated ingress per host. We have the same pattern for the annotations.

File: stable/wordpress/README.md (hunk 1 above)
Comment (timstoop) [reply stable/wordpress/README.md]: We're seemingly converging to use `extraLabels` for such keys. Like `extraAnnotations` and `extraEnv`, etc. I'd recommend renaming this key to `ingress.hosts[0].extraLabels` as well.

Comment (wjeanneau) [reply stable/wordpress/README.md]: Do you think I can rename 'annotations' in the same PR (this is a breaking change).

**Comment by juan131** [comment]
@wjeanneau you need to rebase your PR since it has conflicts with the Chart.yaml on master branch.

@carrodher can you lgtm'd it once it's rebased?

**Pull request closed without merging**